### DISCUSSION TITLE
Request for the Addition ofRTL and LTR Options in @mantine/tiptap

### DISCUSSION THREAD

**Discussion by MonaAghili** [mantinedev/mantine#5285] [Feature requests]
Could you please implement a feature that includes an option button, allowing users to toggle between left-to-right (ltr) and right-to-left (rtl) alignments for a specific selected line in the text? I am specifically referring to the capability of changing the text direction within @mantine/tiptap by selecting a line and clicking a designated button, rather than applying the change to the entire text in the editor.

**Comment by rtivital** [comment]
You can use any Tiptap [extensions](https://tiptap.dev/extensions) with `@mantine/tiptap` package. If the extension that you need does not exist, you can create a custom extension with this functionality. Inline RTL/LTR alignments are a niche feature that is most likely unique to your application, so the code for this feature should live inside your application, not inside the library.

**Reply by shahryarjb** [reply]
Hi @rtivital, I think RTL/LTR button is a global and basic requirement that each editor should have. CkEditor and etc all of them have
but the [extensions](https://tiptap.dev/extensions) thought is very useful approach

Thank you for all your efforts
